### PR TITLE
automod: add basic repost rule

### PR DESCRIPTION
**Pull request by foysalit**
TODO:
- [ ] Verify the threshold values are correct
- [ ] Uncomment ozone report line

This PR adds basic checks for relatively new accounts reposting too frequently and reports them to ozone.

### CODE REVIEW

File: automod/rules/reposts.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	if c.Account.Private != nil {
+		age := time.Since(c.Account.Private.IndexedAt)
+		if age > 30*24*time.Hour {
+			return nil
+		}
+	}
+
+	switch c.RecordOp.Collection {
+	case "app.bsky.feed.post":
+		c.Increment("post", did)
```
Comment (bnewbold) [inline automod/rules/reposts.go:28]: I thought we already had a counter like this elsewhere, but guess we don't!

File: automod/rules/reposts.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+var _ automod.RecordRuleFunc = TooManyRepostRule
+
+// looks for accounts which do frequent reposts
+func TooManyRepostRule(c *automod.RecordContext) error {
+
+	did := c.Account.Identity.DID.String()
+	// Don't bother checking reposts from accounts older than 30 days
+	if c.Account.Private != nil {
+		age := time.Since(c.Account.Private.IndexedAt)
+		if age > 30*24*time.Hour {
```
Comment (bnewbold) [inline automod/rules/reposts.go:21]: I have this as 2 weeks in a couple other places, but 30 days is totally reasonable for this rule.

File: automod/rules/reposts.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"time"
+
+	"github.com/bluesky-social/indigo/automod"
+	"github.com/bluesky-social/indigo/automod/countstore"
+)
+
+var repostDailyThreshold = 500
```
Comment (bnewbold) [inline automod/rules/reposts.go:10]: Definitely curious if we'll have any triggers at this value! I might have started at 100... but maybe these accounts are going really hard already? And there might be heavy organic re-post use by some users.

**Review by bnewbold — APPROVED** [review]

File: automod/rules/reposts.go (hunk 1 above)
Comment (foysalit) [reply automod/rules/reposts.go:28]: I thought we'd have one too but couldn't find one.

File: automod/rules/reposts.go (hunk 3 above)
Comment (foysalit) [reply automod/rules/reposts.go:10]: good point, at least for test, I can set a lower threshold maybe?

File: automod/rules/reposts.go (hunk 2 above)
Comment (foysalit) [reply automod/rules/reposts.go:21]: cool yeah I was going with what Aaron recommended.

**Pull request merged**